vcpkg-configuration.json

```diff
@@ -7,7 +7,7 @@
   "registries": [
     {
       "kind": "git",
-      "repository": "https://github.com/bluequartzsoftware/complex-registry",
+      "repository": "https://github.com/imikejackson/complex-registry",
```

Comment:
Is this supposed to be set to your fork?

Reply:
I just saw your comment

Reply:
At the moment yes. Before I commit the update to the complex-registry I want to make sure this works on all the platforms.